src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+
+        int index = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[index];
```

Comment:
Calling Color.values() inside a loop or a method that is called multiple times can be inefficient, as it creates a new array every time. Consider storing the result of Color.values() in a static final field and reuse it.